src/testRunner/unittests/tsserver/declarationFileMaps.ts

```diff
@@ -308,20 +310,9 @@ namespace ts.projectSystem {
                     kind: ScriptElementKind.functionElement,
                     kindModifiers: "export",
                 },
-                {

```

Comment:
This tests mimics the request of an old VS Code client.  A new client will not pass a file in the default `allOpenProjects` mode, but will in the opt-in `currentProjectMode`.  I didn't think it made sense to return other-project results when the user selected `currentProjectMode` and scoping in this way improved consistency with other scenarios (mostly VS).